Component/Cart/cart.css

```diff
@@ -0,0 +1,70 @@
+.cart-page{
+    margin: 8rem auto;
+}
+
+table{
+    width: 100%;
+    border-collapse: collapse;
+}
+
+.cart-info{
+    display: flex;
+    flex-wrap: wrap;
+}
+
+th{
```

Comment:
Styling with Element names isnt a good practice. Instead you can use a class name, because you never know, you might be using  table again and it might hinder the styling there .

Reply:
th is present  in this component only. will follow what you suggested